### PR TITLE
rc bazel: add crude support for bazel build files

### PR DESCRIPTION
**Pull request by elferdo**

### CODE REVIEW

File: rc/bazel.kak [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# https://docs.bazel.build/versions/master/skylark/build-style.html
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾ 
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .+BUILD %{
```
Comment (lenormf) [inline rc/bazel.kak]: What file is Bazel using? This regex would also match ArchLinux's `PKGBUILD` files, which are shell scripts.

File: rc/bazel.kak [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+# Initialization
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+hook -group bazel-highlight global WinSetOption filetype=bazel_build %{
+        require-module bazel
+
+        add-highlighter window/bazel_build ref bazel_build
+        hook -once -always window WinSetOption filetype=.* %{ remove-highlighter window/bazel_build }
```
Comment (lenormf) [inline rc/bazel.kak]: The indentation here is different from in the detection hook, for some reason.

File: rc/bazel.kak [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+add-highlighter shared/bazel_build regions
+add-highlighter shared/bazel_build/code default-region group
+add-highlighter shared/bazel_build/comment region '#' '$' fill comment
+add-highlighter shared/bazel_build/string region '"' '"' fill string
+
+add-highlighter shared/bazel_build/code/function regex ([\d\w_]+)\( 1:function
+add-highlighter shared/bazel_build/code/operator regex ([=+,()[\]]) 0:operator
+add-highlighter shared/bazel_build/code/variable regex ([\w_][\w\d_]*)\s*= 1:variable
+add-highlighter shared/bazel_build/code/keyword regex (True|False) 1:keyword
+}
```
Comment (lenormf) [inline rc/bazel.kak:36]: Could you add an empty line before the brace, to make it look symmetrical with the opening brace (line 21-22)?

**Review by lenormf — CHANGES_REQUESTED** [review]

File: rc/bazel.kak (hunk 1 above)
Comment (elferdo) [reply rc/bazel.kak]: They're called `BUILD`, I wasn't sure how to deal with it here. Let's see if it is fixed now.

File: rc/bazel.kak (hunk 2 above)
Comment (elferdo) [reply rc/bazel.kak]: Thanks for pointing this out, fixed.

File: rc/bazel.kak (hunk 3 above)
Comment (elferdo) [reply rc/bazel.kak:36]: Done

**Pull request merged**